### PR TITLE
Test PR's on different pulp versions

### PR DESCRIPTION
**Pull request by mdellweg**
This will basically test the same in nightly as in PR's.

[noissue]

### CODE REVIEW

**Comment by mdellweg** [comment]
I like sparse test matrices.

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -16,16 +16,19 @@ jobs:
...
-          - "latest"
-          - "3.8"
-          - "3.7"
+        include:
+          - python: 3.6
+            image_tag: latest
+          - python: 3.7
+            image_tag: 3.8
+          - python: 3.8
+            image_tag: 3.7
```
Comment (daviddavis) [inline .github/workflows/nightly.yml:25]:
```suggestion
        image:
          - "latest"
          - "3.8"
          - "3.7"
        python:
          - "3.6"
          - "3.7"
          - "3.8"
```

**Review by daviddavis — APPROVED** [review]
I'd suggest maybe testing against all combinations for the nightly job but have no objection if we want to keep nightly sparse.

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (mdellweg) [reply .github/workflows/nightly.yml:25]: That would triple the number of tests run. I'd rather not do it.

**Comment by mdellweg** [comment]
Can we mark the issue-checker test required?

**Comment by daviddavis** [comment]
> Can we mark the issue-checker test required?

I can configure the required checks after we merge this.

**Pull request merged**